common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -173,7 +173,7 @@ class BulkOperationsMixin(object):
     mongo_connection.
     """
     def __init__(self, *args, **kwargs):
-        super(BulkOperationsMixin, self).__init__(*args, **kwargs)
+        super(BulkOperationsMixin, self).__init__()
```

Comment:
The only case in which this throws an error is when trying to pass through additional paremeters for the Mongo connection (e.g. replicaSet, readPreference). The reason that this has gone unnoticed is that the default settings in the edx configuration repository don't take advantage of those options. I think that ignoring the `*args`, `**kwargs` is fine as that is part of the reason for accepting additional anonymous arguments. If we remove them from the method signature then it will just break at a different point. I'm happy to be convinced otherwise.